### PR TITLE
Brillig keywords

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by jokva** [comment]
I hoped the tests demo'd it well (apparently not), but you can do `uppercase( returns_string() )`.

Not worth altering the PR obviously, but I'll try to improve the documentation.

**Comment by joakim-hove** [comment]
OK - I'll self merge this tomorrow; unless someone screams!

File: opm/parser/eclipse/EclipseState/tests/Eclipse3DPropertiesTests.cpp [hunk 1]
```diff
@@ -187,11 +187,11 @@ BOOST_AUTO_TEST_CASE(DefaultRegionFluxnum) {
...
-    BOOST_CHECK_THROW(s.props.hasDeckIntGridProperty("NONO"), std::logic_error);
-    BOOST_CHECK_THROW(s.props.hasDeckDoubleGridProperty("NONO"), std::logic_error);
-
-    BOOST_CHECK_THROW(s.props.getIntGridProperty("NONO"), std::logic_error);
-    BOOST_CHECK_THROW(s.props.getDoubleGridProperty("NONO"), std::logic_error);
+//    BOOST_CHECK_THROW(s.props.hasDeckIntGridProperty("NONO"), std::logic_error);
+//    BOOST_CHECK_THROW(s.props.hasDeckDoubleGridProperty("NONO"), std::logic_error);
+//
+//    BOOST_CHECK_THROW(s.props.getIntGridProperty("NONO"), std::logic_error);
+//    BOOST_CHECK_THROW(s.props.getDoubleGridProperty("NONO"), std::logic_error);
```
Comment (pgdr) [inline opm/parser/eclipse/EclipseState/tests/Eclipse3DPropertiesTests.cpp]: Commented-out code?  If they should _no longer throw_ maybe check that they don't?

Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/tests/Eclipse3DPropertiesTests.cpp]: Thank you - no that code should not be commented out. It is back in now.

File: opm/parser/share/keywords/000_Eclipse100/W/WDRILTIM [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{"name" : "WDRILTIM" , "sections" : ["SCHEDULE"], "items": 
+  [{"name" : "WELL"         , "value_type" : "STRING"},
+   {"name" : "DRILL_TIME" , "value_type" : "DOUBLE" , "dimension" : "Time" , "default_value" : 0.0},
+   {"name" : "WORKOVER_CLOSE" , "value_type" : "STRING" , "default_value" : "NO"},
+   {"name" : "COMPARTMENT" , "value_type" : "INT" , "default_value" : 0}]}
+   
+
```
Comment (pgdr) [inline opm/parser/share/keywords/000_Eclipse100/W/WDRILTIM]: Could you fix the two last lines here?

File: opm/parser/share/keywords/000_Eclipse100/W/WLIFT [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ {"name" : "TRIGGER_LIMIT" , "value_type" : "DOUBLE" , "default_value" : 0.0, "comment" : "The dimension here depends on the phase - must be handled in the application"},
+ {"name" : "TRIGGRE_PHASE" , "value_type" : "STRING" , "default_value" : "OIL"},
+ {"name" : "NEW_VFP_TABLE" , "value_type" : "INT"    , "default_value" : 0},
+ {"name" : "NEW_ALQ_VALUE" , "value_type" : "DOUBLE" , "default_value" : 0.0 , "comment" : "The dimension here depends on the phase - must be handled in the application"},
+ {"name" : "NEW_WEFAC"     , "value_type" : "DOUBLE" , "default_value" : 0.0},
+ {"name" : "WWCT_LIMIT"    , "value_type" : "DOUBLE" , "default_value" : 0.0},
+ {"name" : "NEW_THP_LIMIT" , "value_type" : "DOUBLE" , "default_value" : 0.0 , "dimension" : "Pressure"},
+ {"name" : "WGOR_LIMIT"    , "value_type" : "DOUBLE" , "default_value" : 0.0},
+ {"name" : "ALQ_SHIFT"     , "value_type" : "DOUBLE" , "default_value" : 1e20},
+ {"name" : "THP_SHIFT"     , "value_type" : "DOUBLE" , "default_value" : 1e20 , "dimension" : "Pressure"}]}
```
Comment (pgdr) [inline opm/parser/share/keywords/000_Eclipse100/W/WLIFT]: and add a newline at end of file?

And then merge. :)

**Pull request merged**